cmake/onnxruntime_opschema_lib.cmake

```diff
@@ -13,7 +13,7 @@ file(GLOB_RECURSE contrib_ops_schema_src
 
 onnxruntime_add_static_library(ort_opschema_lib ${contrib_ops_schema_src})
 
-set (OPSCHEMA_LIB_DEPENDENCIES onnxruntime_mlas onnxruntime_common onnxruntime_util onnx onnx_proto protobuf::libprotobuf flatbuffers)
+set (OPSCHEMA_LIB_DEPENDENCIES onnxruntime_util onnxruntime_common onnxruntime_mlas onnx onnx_proto protobuf::libprotobuf flatbuffers)
```

Comment:
onnxruntime_mlas should be on the left side of onnxruntime_common. 

Left can depends on right, not vice versa.

Reply:
I have removed all these dependencies. I changed the ordering above based on some existing cmake files, so the existing cmake files may not all be using the same order and may require some fixes. E.g., see: https://github.com/microsoft/onnxruntime/blob/2f2aaf2cf6151e75c4023fc2d0c9d044f5cd212c/cmake/onnxruntime_unittests.cmake#L469